test/tall/regression/0000/0037.stmt

```diff
@@ -30,7 +30,7 @@ remote
     .then((_) => response.first)
     .whenComplete(response.close) // Close when complete.
     .then((sendPort) {
-  sendReceive(sendPort, 'foo').then((msg) {
-    print('remote isolate handler received msg: $msg');
-  });
-});
\ No newline at end of file
+      sendReceive(sendPort, 'foo').then((msg) {
+        print('remote isolate handler received msg: $msg');
+      });
+    });
```

Comment:
Here's an example of a style the old formatter simply couldn't do but which I think looks much better.